kqueue.go

```diff
@@ -256,6 +254,23 @@ func (w *Watcher) addWatch(name string, flags uint32) (string, error) {
 	return name, nil
 }
 
+func (w *Watcher) kq() int {
+	w.mu.Lock()
+	defer w.mu.Unlock()
+	return w._kq
+}
+
+func (w *Watcher) closeKq() {
+	select {
```

Comment:
The select isn't really necessary if there is only a single case (no default case).

Reply:
Fixed in newest rebase.